### ISSUE TITLE
callbackDispatcher not found in Release mode - please add pragma

### ISSUE THREAD

**Issue by ErnestoCuesy** [ABausG/home_widget#116]
Hi and thank you for this fantastic plugin.

I however came across this error when testing my app in Release mode:

`ERROR:flutter/shell/common/shell.cc(93)] Dart Error: Dart_LookupLibrary: library 'package:home_widget/home_widget_callback_dispatcher.dart' not found
`

Everything works fine in Debug and Profile mode.

By looking on the Internet someone suggested adding the vm:entry-point pragma do callbackDispatcher and it works.

```
/// Dispatcher used for calling dart code from Native Code while in the background
@pragma('vm:entry-point')**
void callbackDispatcher() {
  const _backgroundChannel = MethodChannel('home_widget/background');
  WidgetsFlutterBinding.ensureInitialized();

  _backgroundChannel.setMethodCallHandler((call) async {
    final args = call.arguments;

    final callback = PluginUtilities.getCallbackFromHandle(
      CallbackHandle.fromRawHandle(args[0]),
    );

    final rawUri = args[1] as String;

    final uri = Uri.parse(rawUri);

    callback?.call(uri);
  });

```
Could you please include this in future release?

Thank you!

**Comment by tatraef** [comment]
@ErnestoCuesy there is already a fix in home_widget 0.2.0

**Comment by ErnestoCuesy** [comment]
Yes, I just realised today. Thanks!
